common/document_parser/process.py

```diff
@@ -133,29 +138,35 @@ def process_dir(
         parse_func: typing.Callable,
         dir_path: str,
         out_dir: str = "./",
+        thumbnail_dir: str = "thumbnails/",
         meta_data: str = None,
         multiprocess: int = False,
         ocr_missing_doc: bool = False,
-        num_ocr_threads: int = 2
+        num_ocr_threads: int = 2,
+        generate_thumbnails: bool = True
 ):
     """
     Processes a directory of pdf files, returns corresponding Json files
     Args:
         parse_func: Parsing function called on the data
         dir_path: A source directory to be processed.
         out_dir: A destination directory to be processed
+        thumbnail_dir: Where to save the thumbnails
         meta_data: file path of metadata to be processed.
         multiprocess: Multiprocessing. Will take integer for number of cores
         ocr_missing_doc: OCR non-ocr'ed docs in place
         num_ocr_threads: Number of threads used for OCR (per doc)
+        generate_thumbnails: Whether or not to generate png of first page of pdf
     """
 
     p = Path(dir_path).glob("**/*")
     files = [x for x in p if x.is_file() and filetype.guess(str(x)) is not None and (
-        filetype.guess(str(x)).mime == "pdf" or filetype.guess(str(x)).mime == "application/pdf")]
+        filetype.guess(str(x)).mime == "pdf" or filetype.guess(str(x)) != "application/pdf")]
```

Comment:
this line needs to be changed back, the removed line is the correct check

```suggestion
        filetype.guess(str(x)).mime == "pdf" or filetype.guess(str(x)).mime == "application/pdf")]
```